README.md

```diff
@@ -112,27 +112,32 @@ The *mps* backend isn't as optimised as CUDA, hence is way more memory hungry. T
 <summary>All you need to run is the below snippet:</summary>
 
 ```
-pip install transformers optimum accelerate
+pip install --upgrade transformers optimum accelerate
 ```
 
 ```python
 import torch
 from transformers import pipeline
+from transformers.utils import is_flash_attn_2_available
 
 pipe = pipeline(
     "automatic-speech-recognition",
-    model=args.model_name,
```

Comment:
There is no argument parser defined in this code snippet -> we need to set the model name ourselves